### PR TITLE
Adding capabilities to letterhead use gor-icon

### PR DESCRIPTION
**Pull request by tigredonorte**

### CODE REVIEW

File: libs/ui-toolkit/letterhead/src/lib/header/header.component.html [hunk 1]
```diff
@@ -4,9 +4,12 @@
       <ng-content select="[gorLogo]"></ng-content>
     </div>
     <div class="icon-wrapper">
-      <div *ngFor="let iconName of iconList">
-        <img alt="Header logo" *ngIf="!_isPrinting" class="icon" src="/assets/img/icon_{{iconName}}.svg" (click)="onSelectIcon(iconName)">
-      </div>
+      <ng-container *ngIf="!_isPrinting">
+        <div *ngFor="let icon of _iconList">
+          <gor-icon *ngIf="!isIconString" class="icon" [iconFolder]="icon.iconFolder" [icon]="icon" width="24px" height="24px" (click)="onSelectIcon(icon)"></gor-icon>
+          <img *ngIf="isIconString" alt="Header logo" class="icon" src="/assets/img/icon_{{icon}}.svg" (click)="onSelectIcon(icon)">
```
Comment (oliveira-gust) [inline libs/ui-toolkit/letterhead/src/lib/header/header.component.html:10]: Could use `ngIf=...; else`

File: libs/ui-toolkit/letterhead/src/lib/header/header.component.html [hunk 2]
```diff
@@ -4,9 +4,12 @@
       <ng-content select="[gorLogo]"></ng-content>
     </div>
     <div class="icon-wrapper">
-      <div *ngFor="let iconName of iconList">
-        <img alt="Header logo" *ngIf="!_isPrinting" class="icon" src="/assets/img/icon_{{iconName}}.svg" (click)="onSelectIcon(iconName)">
-      </div>
+      <ng-container *ngIf="!_isPrinting">
+        <div *ngFor="let icon of _iconList">
+          <gor-icon *ngIf="!isIconString" class="icon" [iconFolder]="icon.iconFolder" [icon]="icon" width="24px" height="24px" (click)="onSelectIcon(icon)"></gor-icon>
```
Comment (oliveira-gust) [inline libs/ui-toolkit/letterhead/src/lib/header/header.component.html:9]: Hard-coded width and height, was that intended?

Comment (tigredonorte) [reply libs/ui-toolkit/letterhead/src/lib/header/header.component.html:9]: I kept the same width and height that is defined on header.component.css

**Comment by tigredonorte** [comment]
rejecting because the new published version can be used instead of this version

**Pull request closed without merging**